README.md

```diff
@@ -1,2 +1,4 @@
 # code-review-test-repo
 A repo to test code review actions.
+
+A test edit to this file.
```

Comment:
A single test comment on an edit (added via "Add single comment").

Reply:
Forgot to add comment before resolution. Adding now.